src/components/MissingDataPopup/MissingDataPopup.module.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../styles/variables';
+
+.popupWrapper {
+    cursor: pointer;
+    display:inline-block;
+    padding-right: 3.75em;
+}
+
+.popupChoice {
+    display:inline-block;
+    padding:5px;
+    margin:5px 10px 5px 0px;
+    min-width: 3em;
+    cursor:pointer;
+    background: $black;
+    text-align:center;
+    border-radius: 15%;
+    font-weight: 800;
+
+    &:hover{
+        background: #222;
+    }
+
+    &.selected{
+        background: rgb(94, 138, 160);
+    }
+}
+
+.missingDataPopup.missingDataPopup {
+  min-width: 300px;
+  min-height:120px;
+}
+
+.popupContent {
+  display:table-cell;
+}
+
+.footer > div{
+  float:right;
+  padding:5px;
+  min-width: 2.25em;
+  min-height: 2em;
+  cursor:pointer;
+
+  text-align:center;
+  border-radius: 15%;
+  font-weight: 800;
+
+  &.check {
+    background: rgb(56, 167, 117);
+    margin-right: 5px;
+
+    &:hover{
+      background: rgb(77, 196, 142);
```

Comment:
I would defer to @noranda but I think for hover styling we are doing darken 10%

Reply:
Good point.  This had already been changed to darken on our other branch that wasn't pushed, but I'm happy to change or tweak the value.